### PR TITLE
fix: [lw-11523] add advanced recieve on popup view

### PR DESCRIPTION
**Pull request by vetalcore**
# Checklist

- [x] JIRA - [LW-11523](https://input-output.atlassian.net/browse/LW-11523)
- [ ] Proper tests implemented
- [x] Screenshots added.

---

## Proposed solution

Explain how does this PR solves the problem stated in JIRA ticket.
You can also enumerate different alternatives considered while approaching this task.

## Testing

Describe here, how the new implementation can be tested.
Provide link or briefly describe User Acceptance Criteria/Tests that need to be met

## Screenshots

https://github.com/user-attachments/assets/0db60b17-ee53-4e22-a18e-076ca3f63f0e

[LW-11523]: https://input-output.atlassian.net/browse/LW-11523?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by pczeglik-iohk** [comment]
# Allure Report
[`allure-report-publisher`](https://github.com/andrcuns/allure-report-publisher) generated test report!



**processReports**: ✅ [test report](https://lace-qa:8PP5wBaDV6UoXj@dq4ajm5i5q7bz.cloudfront.net/smoke/linux/chrome/7025/11095867946/index.html) for [88ba9948](https://github.com/input-output-hk/lace/pull/1454/commits/88ba99488ecaa76275515ef35949aa32309a92c5)
|       | passed | failed | skipped | flaky | total | result |
|-------|--------|--------|---------|-------|-------|--------|
| Total | 32     | 0      | 4       | 0     | 36    | ✅     |

File: apps/browser-extension-wallet/src/features/receive-info/components/ReceiveInfo.module.scss [hunk 1]
```diff
@@ -6,12 +6,29 @@
   padding: size_unit(4.5) 0;
 }
 
-.title {
-  color: var(--text-color-primary);
-  @include text-heading;
+.subTitle {
```
Comment (szymonmaslowski) [inline apps/browser-extension-wallet/src/features/receive-info/components/ReceiveInfo.module.scss:9]: Could you please merge the two `subTitle` classes?

**Review by szymonmaslowski — APPROVED** [review]
GJ

File: apps/browser-extension-wallet/src/features/receive-info/components/ReceiveInfo.module.scss (hunk 1 above)
Comment (vetalcore) [reply apps/browser-extension-wallet/src/features/receive-info/components/ReceiveInfo.module.scss:9]: .subtitle is not actually used anywhere, removed here dfc095645ccdb5564f53dfd2653de6ef7419d944

**Review by rhyslbw — APPROVED** [review]

**Pull request merged**